framework/py/flwr/cli/run/run.py

```diff
@@ -215,7 +216,8 @@ def _run_with_control_api(
             if is_remote_app:
                 typer.secho(
                     "❌ Failed to start run. Please check that the provided "
-                    "app identifier (@account_name/app_name) is correct.",
+                    "app identifier (@account_name/app_name) is correct, "
+                    "or that it is not using an unsupported gRPC adapter.",
```

Comment:
`unsupported gRCP adapter` might be a bit to technical? When `flwr run ` fails becaues the connection cannot be stablished with a `SuperLink`, [this error message](https://github.com/adap/flower/blob/e7ad112974ba98d6d01990a188ccea14df806b06/framework/py/flwr/cli/utils.py#L484) shown is:

> Connection to the SuperLink is unavailable. Please check your network connection and 'address' in the federation configuration.